src/Domain/RegularGivingMandateRepository.php

```diff
@@ -53,6 +54,14 @@ public function allActiveForDonorWithCharities(PersonId $donor): array
         return $this->getMandatesWithCharities($query);
     }
 
+    /**
+     * @param UuidInterface $uuid
+     * @return RegularGivingMandate|object|null
+     */
+    public function findOneBy(UuidInterface $uuid) {
```

Comment:
Liking the specific method for findOneBy UUID .

Couple of suggestions:
Rename to `findOneByUuid`. PHP doesn't have method overloading so if we want to findOnBy something else that method would need a different name.

Remove the docblock - we can really on native PHP types - we only need to write types types in docblocks when we want to go beyond the capabilities of the native PHP type system.